<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::PreferencesContentsWidget</class>
 <widget class="QWidget" name="Otter::PreferencesContentsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="generalLayout"/>
     </widget>
     <widget class="QWidget" name="contentTab">
      <attribute name="title">
       <string>Content</string>
      </attribute>
      <layout class="QVBoxLayout" name="contentLayout"/>
     </widget>
     <widget class="QWidget" name="privacyTab">
      <attribute name="title">
       <string>Privacy</string>
      </attribute>
      <layout class="QVBoxLayout" name="privacyLayout"/>
     </widget>
     <widget class="QWidget" name="searchTab">
      <attribute name="title">
       <string>Search</string>
      </attribute>
      <layout class="QVBoxLayout" name="searchLayout"/>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="advancedLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
